content/documentation/4-batch-developer-guides/2-batch/1-simple-task.md

```diff
@@ -1,41 +1,148 @@
+
+```java
+{/* highlight-range{2} */}
+@Configuration
+@EnableTask
+public class TaskConfiguration {
+
+    @Autowired
+    private DataSource dataSource;
+
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return args -> {
+            JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
+            jdbcTemplate.execute("CREATE TABLE IF NOT EXISTS " +
+                    "BILL_STATEMENTS ( id int, " +
+                    "first_name varchar(50), last_name varchar(50), " +
+                    "minutes int,data_usage int, bill_amount double)");
+        };
+    }
+}
+```
+
+The `@EnableTask` annotation sets up a TaskRepository which stores information about the task execution such as the start and end time of the task and the exit code.
 
 ### Testing
 
-Show a simple test.
+Now let’s create our test. Update the contents of the [BillsetuptaskApplicationTests.java](https://github.com/cppwfs/edutasksamples/blob/master/billsetuptask/src/test/java/io/spring/billsetuptask/BillsetuptaskApplicationTests.java) with the following code:
+
+```java
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class BillsetuptaskApplicationTests {
+
+	@Autowired
+	private DataSource dataSource;
+
+	@Test
+	public void testRepository() {
+		JdbcTemplate jdbcTemplate = new JdbcTemplate(this.dataSource);
+		int result = jdbcTemplate.queryForObject(
+		"SELECT COUNT(*) FROM BILL_STATEMENTS", Integer.class);
+
+		assertThat(result).isEqualTo(0);
+	}
+}
+```
 
 ## Deployment
 
 Deploy to local, Cloud Foundry and Kubernetes
 
 ### Local
 
-get the jar run it bla
+1. Now let’s take the next step of building the project.
+   From a command line change directory to the location of your project and build the project using maven
+   `mvn clean package`
+
+1. Now let’s execute the application with the configurations required to create the "BILL_STATEMENTS" table in the MySql database.
+   To configure the execution of the billsetuptask application utilize the following arguments:
+
+   1. _spring.datasource.url_ - set the URL to your database instance. In the sample below we are connecting to a mysql `practice` database on our local machine at port 3306.
+   1. _spring.datasource.username_ - the user name to be used for the MySql database. In the sample below it is `root`
+   1. _spring.datasource.password_ - the password to be used for the MySql database. In the sample below it is `password`
+   1. _spring.datasource.driverClassName_ - The driver to use to connect to the MySql database. In the sample below it is `com.mysql.jdbc.Driver'
+
+   ```bash
+   $ java -jar target/billsetuptask-0.0.1-SNAPSHOT.jar \
+   --spring.datasource.url=jdbc:mysql://localhost:3306/practice?useSSL=false \
```

Comment:
can we remove the useSSL flag here?

Reply:
It was required when hooking up to the docker instance of mysql.  Else the connection failed.